Metrics/collect_and_upload.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+xcov --workspace Example/Firebase.xcworkspace --scheme AllUnitTests_iOS --output_directory Metrics --json_report
```

Comment:
It seems like the workspace and scheme could be parameters to this script? Or is it not possible to collect/upload for more than these specific values?

Reply:
It is. Updated to be parameters.